code/UI/Nameplates/PlayerNameplate.scss

```diff
@@ -12,26 +12,36 @@ PlayerNameplate {
 
   .name {
     height: 100%;
-    left: 110px;
     font-family: $primary-font;
+    margin-left: 10px;
     color: white;
     font-size: 45px;
     font-weight: 700;
     text-shadow: 0px 0px 2px 20px rgba(black,1);
     align-items: center;
-    align-content: center;
   }
 
   .avatar {
     margin-left: 8px;
     margin-right: 8px;
-    position: absolute;
     width: 80px;
     height: 80px;
     border-radius: 100px;
     border: 4px solid rgba(white, 0.2);
   }
 
+  .control-glyph {
+    opacity: 0;
+    height: 40%;
+    margin-left: 24px;
+    transition: opacity 0.4s ease-in;
+
+    &.show {
+      transition: opacity 0.4s ease-in;
```

Comment:
i don't think you need to set the transition twice here

Reply:
I didn't think so either, but I tried ease-in-out, ease-out, etc in the show class, to no avail.